src/Mapping/Loader/AnnotationsLoader.php

```diff
@@ -169,7 +169,7 @@ protected function resolveController(?string $controllerPath): callable|false
 
     /**
      * @return PropertySubscription[]
-     * @throws \ReflectionException|\SofaScore\Purgatory\AnnotationReader\ReaderException
+     * @throws \ReflectionException|\Sofascore\PurgatoryBundle\AnnotationReader\ReaderException
```

Comment:
@HypeMC Should we import namespaces that are referenced only in DocBlocks?

Reply:
Personally, I would, but that's out of scope for this one. We can do further code improvements & modernization in a different PR.

Reply:
Ok, so there were only two cases, so I fixed them, but in the other PR to avoid conflicts https://github.com/sofascore/purgatory/pull/55/commits/e6798573286be9591f20537ca048c50b3b8c2926